internal/command/meta_backend.go

```diff
@@ -163,22 +163,22 @@ func (m *Meta) Backend(opts *BackendOpts) (backendrun.OperationsBackend, tfdiags
 		}
 	}
 
-	// If the result of loading the backend is an enhanced backend,
+	// If the result of loading a backend is an operations backend,
 	// then return that as-is. This works even if b == nil (it will be !ok).
 	if enhanced, ok := b.(backendrun.OperationsBackend); ok {
 		log.Printf("[TRACE] Meta.Backend: backend %T supports operations", b)
 		return enhanced, nil
 	}
 
-	// We either have a non-enhanced backend or no backend configured at
-	// all. In either case, we use local as our enhanced backend and the
-	// non-enhanced (if any) as the state backend.
+	// We either have a non-operations backend configured for state storage
+	// or none configured at all. In either case, we use local as our operations backend
+	// and the state-storage backend (if any) to manage state.
 
 	if !opts.ForceLocal {
 		log.Printf("[TRACE] Meta.Backend: backend %T does not support operations, so wrapping it in a local backend", b)
 	}
 
-	// Build the local backend
+	// Build the local operations backend
```

Comment:
Question for reviewer- I wanted to include this comment but wasn't sure if it added context that couldn't already be learned from the code. Does this comment help with understanding, or does it add clutter?

```suggestion
	// Build the local operations backend
	// Note: If b is nil then we are using implicit local state storage.
```